oidc_client/utils/utils.py

```diff
@@ -0,0 +1,133 @@
+"""General Utility Functions."""
+
+from uuid import uuid4
+
+import aiohttp
+
+from aiohttp import web
+
+from ..config import CONFIG
+from .logging import LOG
+
+
+def ssl_context():
```

Comment:
We can have a placeholder for the necessary things here even just returning `None` e.g.:
```
import ssl
sslcontext.load_cert_chain(ssl_certfile, ssl_keyfile)
sslcontext = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
sslcontext.check_hostname = False
```

and documenting how people can set up this.

Reply:
described in configuration file